docs/operations.md

```diff
@@ -0,0 +1,160 @@
+  namespace: default
+spec:
+  storageClassName: "local-ssd"
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 100Gi
+```
+
+#### ReplicaSet
+
+```
+apiVersion: v1
+kind: ReplicaSet
+metadata:
+  name: my-cluster-0
+  namespace: default
+spec:
+  podTemplate:
+    volumes:
+      - name: etcd-storage
+        persistentVolumeClaim:
+          claimName: my-cluster-0
+    containers:
+      - name: etcd
+        volumeMounts:
+          - mountPath: "/var/lib/etcd"
+            name: etcd-storage
+```
+
+Kubernetes will then dynamically provision a ``PersistentVolume`` of 100Gi backed by an SSD,
+and "bind" that PV to the PVC above.
+(or it will bind per-provisioned PV to that PVC).
+
+Kubernetes will also create a ``Pod`` based on the ``ReplicaSet`` template above.
+
+This ``Pod`` will remain unscheduled until the PVC is bound to a PV.
+It will then be scheduled and started on the same Kubernetes node as the PV.
+
+Eventually, you will have an empty Etcd cluster of three nodes, with a total of 300Gi SSD storage available.
+
+### Reboot a Kubernetes Node
+
+Assuming you are using [Local Persistent Volumes](https://kubernetes.io/docs/concepts/storage/volumes/#local),
+which are contstrained to a particular Kubernetes node.
+how do you **reboot** a Kubernetes node, without disrupting the ``EtcdCluster`` ??
+
+**NOTE:** This operation   **does not**  require the ``improbable-eng/etcd-cluster-operator``to be running.
+The Etcd peer ``ReplicaSets`` ensure that the cluster can function even without the operator which created them.
+
+**NOTE:** Ensure that your ``EtcdCluster`` has sufficient peers on other Kubernetes nodes, to maintain quorum.
+
+1. Reboot the Kubernetes node
+   1. Kubernetes will delete the current `Pod` (it will be scheduled for deletion), but the PVC, the PV, will remain.
+   1. Kubernetes ``ReplicaSet`` controller will ensure that a new ``Pod`` is created, but it will remain un-schedulable until the Kubernetes node reboots.
+   1. This is because the Kubernetes Scheduler knows that the `Pod` placement is constrained by the location of the existing PVC and bound PV.
+   1. When the server has rebooted, Kubernetes will be able to schedule the ``Pod`` it will start up, with the existing data and rejoin the Etcd cluster.
+
+### Decommission a Kubernetes Node that has an EtcdPeer
+
+Assuming you are using [Local Persistent Volumes](https://kubernetes.io/docs/concepts/storage/volumes/#local),
+which are contstrained to a particular Kubernetes node.
+how do you **decommission** a Kubernetes node, without disrupting the ``EtcdCluster``?
+
+**NOTE:** This operation  **requires** the ``improbable-eng/etcd-cluster-operator``to be running.
+
+**NOTE:** Ensure that your ``EtcdCluster`` has sufficient peers on other Kubernetes nodes, to maintain quorum.
+
+1. Add a new Kubernetes node somewhere that supports the StorageClass used by Etcd.
+1. Cordon the old Kubernetes node.
+1. Scale out the Etcd cluster by one.
```

Comment:
This is probably wrong. See https://github.com/etcd-io/etcd/blob/master/Documentation/faq.md#should-i-add-a-member-before-removing-an-unhealthy-member